bidscoin/bids.py

```diff
@@ -1869,25 +1873,64 @@ def insert_bidskeyval(bidsfile: Union[str, Path], bidskey: str, newvalue: str, v
     return newbidsfile
 
 
+def remove_run_keyval(bidsname: str) -> str:
```

Comment:
I think this function is not needed and can be achieved by calling `insert_keyval(bidsname, 'run', '', False)`